### ISSUE TITLE
Conflict with Hide Top Bar 

### ISSUE THREAD

**Issue by stpnwf** [brunodrugowick/notification-position-gnome-extension#14]
Notifications are squished if I have [Hide Top Bar](https://gitlab.gnome.org/tuxor1337/hidetopbar)'s Intellihide option "Only hide panel when a window takes the space" enabled.
![Screenshot from 2023-04-18 10-00-30](https://user-images.githubusercontent.com/81482539/232792232-ee8aa34f-ff65-40ff-9fa9-46c062202b1e.png)

**Comment by brunodrugowick** [comment]
I can't reproduce this. I installed the mentioned extensions and got notifications normally.
![working_with_hidetopbar](https://github.com/brunodrugowick/notification-position-gnome-extension/assets/7481200/aa36275d-a553-4daf-84ea-ae0e31a47ea9)

The thing you point to in the screenshot looks like a scroll bar. 🤔

**Comment by stpnwf** [comment]
Sorry, I had just turned off the extension Notification Banner Reloaded and turned this extension on to test if it had the same issue. I guess I needed to log out to test it. I retested and the issue does not happen.
On Notification Banner Reloaded , it only happens when the animation is set to Slide from Right. 
Since this extension's default animation is Slide from Top, it doesn't happen.
Sorry for the confusion.
